src/Operation/Count.php

```diff
@@ -162,6 +162,39 @@ public function execute(Server $server)
         return (integer) $result->n;
     }
 
+    /**
+     * Explain the operation.
+     *
+     * @see Explainable::explain()
+     * @param Server $server
+     * @param $command
+     * @param array $options
+     * @return array|object
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function explain($server, $command, $options = [])
+    {
+        if ( ! isset($options['verbosity'])) {
+            $options['verbosity'] = 'allPlansExecution';
+        }
+
+        $cmd = new Command(['explain' => ['count' => $this->collectionName, 'query' => $command], 'verbosity' => $options['verbosity']]);
```

Comment:
The value of `explain` here will need to be the entire document used to create the `MongoDB\Driver\Command` object in `createCommand()` below. Specifically, the [constructor's](http://php.net/manual/en/mongodb-driver-command.construct.php) first argument.